docs/NewSite.md

```diff
@@ -9,20 +9,32 @@ and the url appropriately.
 
 ## Establish domains with Stanford
 1. Add the domains to the [NetDB record](https://netdb.stanford.edu/node_info?name=swshumsci.stanford.edu&history=%252Fqsearch%253Fsearch_string%253Dswshumsci%2526search_type%253DNodes)
-    * Keep with the pattern: [sitename]-dev, [sitename]-stage, and [sitename]-prod
+  * Keep with the pattern: [site-name]-dev, [site-name]-stage, and [site-name]-prod
 1. Wait for the DNS Refresh. This occurs every half hour at :05 and :35 past the hour.
 1. Ensure the Vhost points to Acquia by pinging the url. `ping newvhost.stanford.edu`
 
-## Site Directory and settings.
-1. Disable shield on all sites on the dev and test environments. this will help when the LE certs are generated. `blt drupal:module:uninstall shield [environment:dev,test]`
+## Setup Domains and SSL Certs in Acquia
+1. Disable the shield module for all sites on the dev and test environments. This ensures the the SSL certs can be verified when they are issued.
+  * `blt drupal:module:uninstall shield dev`
+  * `blt drupal:module:uninstall shield stage`
+1. Add the new domains to each of the Acquia environments (dev, stage, and prod). You can do this through the Acquia UI, or by using `blt humsci:add-domain`. For example:
+  * `blt humsci:add-domain dev [site-name]-dev.stanford.edu`
+  * `blt humsci:add-domain test [site-name]-test.stanford.edu`
+  * `blt humsci:add-domain prod [site-name]-prod.stanford.edu`
+1. Re-issue the SSL certs for each environment. For example:
+  * `blt humsci:letsencrypt:add-domain --domains=[site-name]-dev.stanford.edu -- dev`
+  * `blt humsci:letsencrypt:add-domain --domains=[site-name]-stage.stanford.edu -- test`
+  * `blt humsci:letsencrypt:add-domain --domains=[site-name]-prod.stanford.edu -- prod`
+1. Get the newly built SSL certs for each environment (`blt humsci:letsencrypt:get-cert [environment]`) and add them to their corresponding [Acquia Dashboards](https://cloud.acquia.com/app/develop/applications/23a85077-2967-41a4-be22-a84c24e0f81a/environments/265865-23a85077-2967-41a4-be22-a84c24e0f81a/ssl) by clicking "Install SSL Certificate" and filling out the details.
```

Comment:
This is no longer needed with the command `blt humsci:update-cert [env]` command

Reply:
Nice! Updated 👍.